src/files.js

```diff
@@ -559,7 +559,7 @@ FOAM_FILES([
   { name: "foam/u2/view/ReadManyToManyRelationshipPropertyView", flags: ['web'] },
   { name: "foam/u2/view/ReferenceArrayView", flags: ['web'] },
   { name: "foam/u2/view/DraftDetailView", flags: ['web'] },
-  { name: "foam/u2/view/FObjectArrayView", flags: ['web'] },
+  { name: "foam/u2/view/TitledArrayView", flags: ['web'] },
```

Comment:
Why delete the old one?

Reply:
Can't we have both?

Reply:
We could. The view itself doesn't add much in terms of functionality so I removed it.

Reply:
Can you add it back to files.js?

Reply:
Already there